### PR TITLE
Fix position/orientation results

### PR DESCRIPTION
**Pull request by GerhardHippmann**
- Enable position and orientation result signals for all user-defined Object3Ds, independently of visualisation/animation configuration
- Replace orientation result matrix R_abs -> Cardan angles rot123_abs

### CODE REVIEW

File: src/Composition/object3D.jl [hunk 1]
```diff
@@ -632,19 +633,23 @@ hasChildJoint(        obj::Object3D{F}) where F <: Modia3D.VarFloatType = obj.ha
...
-isVisible(feature::Modia3D.AbstractObject3DFeature, exportAnimation::Bool) = false
-isVisible(feature::Modia3D.AbstractScene, renderer::Modia3D.AbstractRenderer) = false
-isVisible(feature::Modia3D.AbstractScene, exportAnimation::Bool) = false
+function isVisible(obj::Object3D{F})::Bool where F <: Modia3D.VarFloatType
+    if isSolid(obj) || isVisual(obj)
+        return !isnothing(obj.feature.visualMaterial) && !isnothing(obj.feature.shape)
+    else
+        return false
+    end
+end
```
Comment (AndreaNeumayr) [inline src/Composition/object3D.jl:652]: Please, check if a default return type is needed outside of if/else with (code_warntype).

File: src/AnimationExport/exportAnimation.jl [hunk 2]
```diff
@@ -436,8 +436,8 @@ end
 
 
 function exportAnimation(scene)
```
Comment (AndreaNeumayr) [inline src/AnimationExport/exportAnimation.jl:438]: Please, define return type and scene::Scene{F} in function.

File: src/Composition/object3D.jl (hunk 1 above)
Comment (GerhardHippmann) [reply src/Composition/object3D.jl:652]: 'code_warntype' does not report an issue with the explicitly defined return type 'Bool'.

**Comment by GerhardHippmann** [comment]
@AndreaNeumayr  Bug in dependencies has been fixed. So from my point of view we can release now. Please do not forget to update release notes, version numbers and dates.

**Pull request merged**